admin/modules/config/post_icons.php

```diff
@@ -114,7 +116,8 @@
 			}
 
 			$path = $mybb->input['pathfolder'];
-			$dir = @opendir(MYBB_ROOT.$path);
+			$dir = !empty(MYBB_ROOT.$path) && is_dir(MYBB_ROOT.$path) && is_readable(MYBB_ROOT.$path) ? opendir(MYBB_ROOT.$path) : null;
```

Comment:
(`empty()` doesn't seem to support non-variable expressions on PHP <= 5.5 - one of the reasons why the lint test is failing)